docs/en/operation/User-CLI.md

```diff
@@ -597,6 +607,16 @@ The `--replication` flag can be used to load the data into multiple workers.
 $ ./bin/alluxio fs distributedLoad --replication 2 /data/today
 ```
 
+### distributedMv
+
+The `distributedMv` command moves a file or directory in the Alluxio file system distributed across workers.
+
+If the source designates a directory, `distributedMv` moves the entire subtree at source to the destination.
```

Comment:
Does distributed move work across mount points? If not, we should mention that here.

Reply:
I don't actually know but I believe it does.

Reply:
Based on the code it looks like it does a copy and a delete which should be fine.